neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -215,5 +217,24 @@ public virtual bool Verify(Snapshot snapshot, IEnumerable<Transaction> mempool)
             if (net_fee < 0) return false;
             return this.VerifyWitnesses(snapshot, net_fee);
         }
+
+        public StackItem ToStackItem()
+        {
+            return new VM.Types.Array
+            (
+                new StackItem[]
+                {
```

Comment:
@neo-project/core What should be the order?

Reply:
Same as they are in `Transaction`.